### DISCUSSION TITLE
More Configurable Items: Such as movement keymap

### DISCUSSION THREAD

**Discussion by mistgc** [darrenburns/posting#147] [Ideas]
I am a Vim user while I am also a Colemak keyboard layout user. So that I do not use "HJKL" , I use "HENI" instead.

I wish that the Movement Keymap can be configurable.

Colemak Layout like this:

![Colemak2](https://github.com/user-attachments/assets/54416352-b8e9-4400-b077-33cb048609b6)

**Comment by darrenburns** [comment]
Thanks for the suggestion. I'm going to add support for configuring the jump mode keys in the future.

**Comment by ShawnMcCool** [comment]
I am in the same situation, I use vim navigation for _everything_ but I have a different keyboard layout than the us qwerty. Being able to rebind navigation like left/right up/down would be HUGE for me.
